src/main/java/bio/terra/common/logging/GoogleJsonLayout.java

```diff
@@ -133,11 +141,21 @@ public void stop() {
       for (Object arg : event.getArgumentArray()) {
         try {
           if (arg instanceof Map) {
+            // Handle arbitrary Map by splatting each key-value pair into the main output map.
             Map<String, Object> jsonMap = (Map<String, Object>) arg;
             jsonMap.forEach(map::put);
           } else if (arg instanceof JsonNode) {
+            // Handle Jackson JsonNode by splatting each property sub-tree into the main output map.
             JsonNode jsonNode = (JsonNode) arg;
             jsonNode.fields().forEachRemaining(entry -> map.put(entry.getKey(), entry.getValue()));
```

Comment:
Do object/composite/nested properties get expanded into JSON as well here? Seems like you'd have to recurse through more than on level. Looks like the value here is another JsonNode according to the docs.